README.md

```diff
@@ -258,3 +258,79 @@ Ability to hotplug block devices has been added, which makes it possible to use
 ## How to check if container uses devicemapper block device as its rootfs
 
 Start a container. Call mount(8) within the container. You should see '/' mounted on /dev/vda device.
+
+# Devices
+
+Support has been added to pass [VFIO](https://www.kernel.org/doc/Documentation/vfio.txt) 
+assigned devices on the docker command line with --device.
+Support for passing other devices including block devices with --device has
+not been added added yet.
+
+## How to pass a device using VFIO-passthrough
+
+1. Requirements
+
+IOMMU group represents the smallest set of devices for which the IOMMU has
+visibility and which is isolated from other groups.  VFIO uses this information
+to enforce safe ownership of devices for userspace. 
+
+You will need Intel VT-d capable hardware. Check if IOMMU is enabled in your host
+kernel by verifying `CONFIG_VFIO_NOIOMMU` is not in the kernel config. If it is set,
+you will need to rebuild your kernel.
+
+The following kernel configs need to be enabled:
+```
+CONFIG_VFIO_IOMMU_TYPE1=m 
+CONFIG_VFIO=m
+CONFIG_VFIO_PCI=m
+```
+
```

Comment:
@amshinde you also need to add the requirement that iommu needs to be enabled at boot 
```
intel_iommu=on
```
Most distro's do not enable iommu by default